doc/changelog/02-specification-language/10331-minim-prop-toset.rst

```diff
@@ -0,0 +1,3 @@
+- Changed heuristics for universe minimization to :g:`Set`: also use
```

Comment:
```suggestion
- **Changed:** Heuristics for universe minimization to :g:`Set`: also use
```

Reply:
It actually is closed to what the refman says: only "fresh" universes are minimized